ecosystem/resources/members/prototype-quantum-kernel-training.toml

```
@@ -1,6 +1,6 @@
 name = "Quantum Kernel Training"
 url = "https://github.com/qiskit-community/prototype-quantum-kernel-training"
-description = "The quantum kernel training (QKT) toolkit is designed to enable users to leverage quantum kernels for machine learning tasks; in particular, researchers who are interested in investigating quantum kernel training algorithms in their own research, as well as practitioners looking to explore and apply these algorithms to their machine learning applications."
+description = "Use quantum kernels for machine learning. For researchers and practitioners looking to explore and apply these algorithms to their work."
```

```suggestion
description = "Allows machine learning researchers and practitioners to use quantum kernels for these algorithms."
```

---

I don't know about that suggestion, but that second sentence "For researchers..." is currently a fragment.

---

Thanks! How about this?

```suggestion
description = "Use quantum kernels for machine learning. This toolkit is for researchers and practitioners looking to explore and apply these algorithms to their work."
```